src/02-data-types.js

```diff
@@ -95,7 +111,13 @@ function checkIfCharacterIsInString(text, character) {}
  *  isEven(11);
  *  //> false
  */
-function isEven(num) {}
+function isEven(num) {
+  if (num % 2 == 0) {
```

Comment:
We should always aim to use. ` === ` in our comparison.  While we don't have any chance here of a string of zero in this instance its good to know that ` 0 == "0" ` would return true.